### PR TITLE
Update Email Connector example doc

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
This PR includes updates to the below mentioned page.
https://mi.docs.wso2.com/en/4.3.0/reference/connectors/email-connector/email-connector-example/ 

The updated page will appear as below. 
Screenshot 2024-07-09 at 11 29 51
Screenshot 2024-07-09 at 11 30 12
Screenshot 2024-07-09 at 11 30 24
Screenshot 2024-07-09 at 11 30 43
Screenshot 2024-07-09 at 11 30 57
Screenshot 2024-07-09 at 11 31 12
Screenshot 2024-07-09 at 11 31 27
Screenshot 2024-07-09 at 11 31 41
Screenshot 2024-07-09 at 11 31 54
Screenshot 2024-07-09 at 11 32 07
Screenshot 2024-07-09 at 11 32 20
Screenshot 2024-07-09 at 11 35 25
Screenshot 2024-07-09 at 11 35 42
Screenshot 2024-07-09 at 11 35 55
Screenshot 2024-07-09 at 11 36 06
Screenshot 2024-07-09 at 11 36 16
Screenshot 2024-07-09 at 11 36 27
Screenshot 2024-07-09 at 11 36 39
Screenshot 2024-07-09 at 11 36 50
Screenshot 2024-07-09 at 11 37 00
Screenshot 2024-07-09 at 11 37 44
Screenshot 2024-07-09 at 11 37 58
Screenshot 2024-07-09 at 11 38 11
Screenshot 2024-07-09 at 11 38 27
Screenshot 2024-07-09 at 11 38 38
Screenshot 2024-07-09 at 11 38 46
Screenshot 2024-07-09 at 11 38 55

### CODE REVIEW

File: en/docs/reference/connectors/email-connector/email-connector-example.md [hunk 1]
```diff
@@ -18,22 +18,23 @@ The example consists of an API named as EmailConnector API with two resources `s
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/email-connector/email-connector-example.md:21]:
```suggestion
## Setting up the Integration Project
```

Let's use this format to ensure consistency of the titles across the page.

Comment (chathuranga-jayanath-99) [reply en/docs/reference/connectors/email-connector/email-connector-example.md]: Sure

File: en/docs/reference/connectors/email-connector/email-connector-example.md [hunk 2]
```diff
@@ -18,22 +18,23 @@ The example consists of an API named as EmailConnector API with two resources `s
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setting up the Integration Project
```
Comment (chanikag) [inline en/docs/reference/connectors/email-connector/email-connector-example.md:21]:
```suggestion
## Set up the Integration Project
```

File: en/docs/reference/connectors/email-connector/email-connector-example.md [hunk 3]
```diff
@@ -18,22 +18,23 @@ The example consists of an API named as EmailConnector API with two resources `s
...
 
-Follow these steps to set up the Integration Project and the Connector Exporter Project. 
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
```
Comment (chanikag) [inline en/docs/reference/connectors/email-connector/email-connector-example.md:23]:
```suggestion
Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the **Integration Project**. 
```

File: en/docs/reference/connectors/email-connector/email-connector-example.md [hunk 4]
```diff
@@ -57,20 +59,25 @@ Follow these steps to set up the Integration Project and the Connector Exporter
...
     - Password - &lt;your_email_password&gt; 
     > **NOTE**: If you have enabled 2-factor authentication, an app password should be obtained as instructed [here](https://support.google.com/accounts/answer/185833?hl=en).
+    - Require TLS - false
 
-    <img src="{{base_path}}/assets/img/integrate/connectors/email-conn-4.png" title="Connection parameters." width="400" alt="Connection parameters."/>
+    <img src="{{base_path}}/assets/img/integrate/connectors/email-conn-add-smtps-conn.png" title="Connection parameters." width="400" alt="Connection parameters."/>
     
-7. After the connection is successfully created, select the created connection as 'Connection' from the drop down in the properties window.
-   <img src="{{base_path}}/assets/img/integrate/connectors/email-conn-5.png" title="Selecting the connection." width="800" alt="Selecting the connection."/>
+7. After the connection is successfully created, select the created connection as 'Connection' from the drop down in appeared window after going into the `email` connector and selecting `send` operation. <br/>
```
Comment (chanikag) [inline en/docs/reference/connectors/email-connector/email-connector-example.md:79]:
```suggestion
7. After the connection is successfully created, select the created connection as `Connection` from the drop down in appeared window after going into the `email` connector and select `send` operation. <br/>
```

File: en/docs/reference/connectors/email-connector/email-connector-example.md [hunk 5]
```diff
@@ -101,22 +107,22 @@ Follow these steps to set up the Integration Project and the Connector Exporter
...
+    <img src="{{base_path}}/assets/img/integrate/connectors/email-conn-add-get-email-body.png" title="Adding getEmailBody operation." width="800" alt="Adding getEmailBody operation."/>
 
     > **NOTE**: Further, you can use `getAttachment` operation to retrieve attachment content if there are any. Refer [Reference Documentation](email-connector-config/) to learn more.
 
-16. Next, we will use a [Payload Factory Mediator]({{base_path}}/reference/mediators/payloadfactory-mediator/), to add the email content to the same response we received from `list` operation and configure the Payload mediator as shown below.
-    <img src="{{base_path}}/assets/img/integrate/connectors/email-conn-11.png" title="Adding payload factory mediator." width="800" alt="Adding payload facotry mediator."/>
+16. Next, we will use a [Payload Mediator]({{base_path}}/reference/mediators/payloadfactory-mediator/), to add the email content to the same response we received from `list` operation configuring the Payload mediator as shown below.
```
Comment (chanikag) [inline en/docs/reference/connectors/email-connector/email-connector-example.md:122]:
```suggestion
16. Next, we will use a [Payload Factory Mediator]({{base_path}}/reference/mediators/payloadfactory-mediator/), to add the email content to the same response we received from `list` operation configuring the Payload mediator as shown below.
```

File: en/docs/reference/connectors/email-connector/email-connector-example.md [hunk 6]
```diff
@@ -127,13 +133,16 @@ Follow these steps to set up the Integration Project and the Connector Exporter
     </email>
     ```
     
+    You can add every parameter following simillar steps given below. <br/>
+    <img src="{{base_path}}/assets/img/integrate/connectors/email-conn-payload-add-param.png" title="Adding payload parameter." width="300" alt="Adding payload parameter."/>
+
     Here, you may observe that we are obtaining `TEXT_CONTENT` property which is being set when getEmailBody is invoked to retrieve the email content. You can find the list of similar properties set in this operation [here]({{base_path}}/reference/connectors/email-connector/email-connector-config/).
```
Comment (chanikag) [inline en/docs/reference/connectors/email-connector/email-connector-example.md:139]:
```suggestion
    Here, you may observe that we are obtaining `TEXT_CONTENT` property which is set when getEmailBody is invoked to retrieve the email content. You can find the list of similar properties set in this operation [here]({{base_path}}/reference/connectors/email-connector/email-connector-config/).
```

**Review by chanikag — APPROVED** [review]

**Review by chanikag — APPROVED** [review]

**Pull request merged**